main.tf

```
@@ -37,45 +37,45 @@ resource "google_cloud_scheduler_job" "job" {
  *****************************************/
 
 module "pubsub_topic" {
-  source     = "github.com/terraform-google-modules/terraform-google-pubsub?ref=v0.1.0"
-  topic      = "${var.topic_name}"
-  project_id = "${var.project_id}"
+  source     = "github.com/terraform-google-modules/terraform-google-pubsub?ref=v1.0.0"
```

This should refer to the registry.

```suggestion
  source     = "terraform-google-modules/pubsub/google"
  version    = "~> 1.0"
```